.prow/features.yaml

```diff
@@ -308,3 +308,39 @@ presubmits:
               cpu: 2
             limits:
               memory: 6Gi
+
+  - name: pre-kubermatic-encryption-at-rest-e2e
+    run_if_changed: "(pkg/controller/seed-controller-manager/encryption-at-rest-controller/|pkg/test/e2e/encryption-at-rest/)"
```

Comment:
I'm wondering if this should be run more frequently than with those files, e.g. when we make changes to the Kubernetes version or the apiserver.

Reply:
Every commit might be a bit much, but I think it makes sense to test it against a) any apiserver changes and b) any Kubernetes version changes (which is somewhere in `pkg/defaulting`, afaik?).

Reply:
Sounds good! I’m planning to run this test case if there are changes in `pkg/defaulting` or `pkg/resources`. Does that approach work for you?

Reply:
I think `pkg/resources/apiserver` should be sufficient, otherwise lgtm!

Reply:
Thanks for the review. Updated accordingly in 238d6da05ff1a1f2b0a28bb605a3200635034771

Reply:
Sorry, I wasn't clear in what I meant to say - your previous triggers were perfect, the addition of `pkg/defaulting` makes sense, I just meant to say instead of `pkg/resources` you can "just" add `pkg/resources/apiserver`. Not replace everything with it, but I can see how it could be read that way.

Reply:
oh, my bad. i misunderstood, thanks for the clarification :) 
i've updated the PR accordingly.